### DISCUSSION TITLE
Persistence of Fluttermoji data

### DISCUSSION THREAD

**Discussion by Hami0095** [psk907/fluttermoji#33] [Q&A] [answered]
I want to ask, whether the FluttermojiAvatar data saved by the user persist it's state or not? Or as a developer we have to manage it? 
I'm confused because I have seen shared_preference dependency in the pub.dev package page of this package

**Comment by psk907** [comment, marked as answer]
The Fluttermoji saved by the user is persisted across app launches. As you've noticed, shared_preferences is used to achieve this behaviour.

When the user presses on the save button that is configured with your customized widget, the avatar configuration is persisted.
